### ISSUE TITLE
Colony no longer exists. No colonists, no ID, nothing but empty buildings.

### ISSUE THREAD

**Issue by Bihlbo** [ldtteam/minecolonies#2232]
### Minecolonies version

Minecraft 1.12.2
Forge 14.23.1.2604
The game is the MineColonies Official modpack accessed through the Twitch app. Version 4.19

### Actual behaviour

Was playing, had about 25 colonists in a town 63 chunks in area. Game crashed. Nothing unusual lead up to the crash. No reason was given by Minecraft. It simply froze, shut down and said it crashed. When I reloaded the game, no colonists exist in the world. None of the hut chests can be interacted with by right-clicking. Results of the following commands:

/mc colonies list - the list format shows, with no content. No colony listed.
/mc colony info # - numbers -15 through 15 all return the same message, that "Colony with ID X not found", except for the number -1 and when no number is entered. That returns "Colony with mayor -1 not found."
/mc colony refresh - same as above.
/mc colony addOfficer - "Please define a colony or player" is returned.
/mc colony ownerchange - same as above.
/mc colony info Bihlbo - "Colony with mayor Bihlbo not found" is returned.
/mc citizens info - "Please define a valid citizen and/or colony" is returned.
/mc citizens respawn - same as above.
/mc citizens kill - same as above.
/mc citizens list - in red: "An unknown error occurred while attempting to perform this command" is returned.

### Steps to reproduce the problem

1. As I don't know what caused the crash, I cannot reproduce it. However, this has happened a few times before (4-6 times I'm guessing). No specific action has tied the events together. It has happened with 4 starting colonists and two buildings, and now this at the other end of the extreme. In the past I just broke and replaced the town hall and other hut chests, causing a new colony to be built in the same space as the old. Now however I'm looking at replacing all the advanced buildings with level 1 versions then very slowly advancing through the levels by destroying what's there, which I don't think is viable.

As long as it's possible to lose a colony completely by the game simply crashing, it's hard to call this modpack playable, so I really hope this helps.

[crash-2018-02-18_01.28.15-client.txt](https://github.com/Minecolonies/minecolonies/files/1734615/crash-2018-02-18_01.28.15-client.txt)
[crash-2018-02-18_01.28.15-server.txt](https://github.com/Minecolonies/minecolonies/files/1734616/crash-2018-02-18_01.28.15-server.txt)

**Comment by Raycoms** [comment]
Probably your colony got lost, please load a backup of your colonyX.dat
file in world/minecolonies.


2018-02-18 11:02 GMT+00:00 Bill Wiltfong <notifications@github.com>:

**Comment by Bihlbo** [comment]
That worked. But oh expletives, is it disheartening to lose a colony.

**Comment by mcDandy** [comment]
It happened to me, but without crash. I vaited for about an hour to ae2 crystal seeds to grow. after restart I noticed that I cannot acces or place any bulidings.

**Comment by Raycoms** [comment]
@mcDandy what version are you playing on?

**Comment by atxmatt2510** [comment]
I am playing on the release ( minecolonies-1.12.2-0.10.109-RELEASE-universal ) and I'm experiencing this issue with no crash. 
Playing on a forge server ( forge-1.12.2-14.23.5.2768-universal.jar ) 

Seems like it hasn't been working for a little while. Could have stopped working when I updated to the beta, however I have rolled back to the release. I have 69 backups and none of them work. All of them have a colonies.dat and colony1_0.dat
